docs/Installation.rst

```diff
@@ -191,12 +191,29 @@ If you prefer using Python Package Index, you can retrieve all Python dependenci
 Installing CMS
 ==============
 
+In order to run CMS there are some preparation steps to run (like installing the sandbox, compiling localization files, creating the ``cmsuser``, and so on). You can either do all these steps by hand or you can run the following command:
+
+.. sourcecode:: bash
+
+    sudo ./prerequisites.py install
+
 You can download CMS |release| from :gh_download:`GitHub` and extract it on your filesystem. After that, you can install it (recommended, not necessary though):
 
 .. sourcecode:: bash
 
-    ./setup.py build
-    sudo ./setup.py install
+    ./setup.py install --user
+
+Or, if you prefer to use pip:
+
+.. sourcecode:: bash
+
+    pip install --user .
+
+.. note::
+
+    If you are going to use a `virtual environment <https://virtualenv.pypa.io/en/latest/>`_ then you won't need the ``--user`` flag.
```

Comment:
I'll create a separate section then (like the ones for Ubuntu, Archlinux, Debian). But maybe that logically fits better in a separate PR...